### DISCUSSION TITLE
Define the data of the pictures to be shared

### DISCUSSION THREAD

**Discussion by racecrew** [photoprism/photoprism#3678] [Ideas]
Hello,

my use case is: "I take pictures with location tags, want to see the map of the pictures in PhotoPrism, but sometimes I don't want to share this info within the picture"

What do you think, would it make sense to have a feature to be able to pre-define what kind of data should be shared? E.g. don't sahre location tags etc, or don't show it when somebody access the pictures via shared link. 

The pre-definition could be done here:

![grafik](https://github.com/photoprism/photoprism/assets/17234283/fb461dd0-e165-4796-ac86-56791b055ea4)

Thanks!

**Comment by graciousgrey** [comment]
We do have some related feature requests :) Feel free to leave a :+1: in there:
- https://github.com/photoprism/photoprism/issues/406
- https://github.com/photoprism/photoprism/issues/557

**Reply by racecrew** [reply]
Thanks for the reply. I read the requests, this is exactly what I have mentioned in my question 👍
